RELEASES.md

```diff
@@ -6,8 +6,87 @@ Unreleased.
 
 ### Added
 
+* Allow the MPL-2.0 and OpenSSL licenses in dependencies of wasmtime.
+  [#6136](https://github.com/bytecodealliance/wasmtime/pull/6136)
+
+* The initial integration between winch and wasmtime has landed behind a cargo
+  feature.
+  [#6119](https://github.com/bytecodealliance/wasmtime/pull/6119)
+
+* Add a bounds-checking optimization for dynamic memories and guard pages.
+  [#6031](https://github.com/bytecodealliance/wasmtime/pull/6031)
+
+* Add support for generating perf maps for simple perf profiling.
+  [#6030](https://github.com/bytecodealliance/wasmtime/pull/6030)
```

Comment:
Could this mention that the old `--jitdump` and `--vtune` flags are now `--profile vtune` now? (e.g. stuff is now behind `--profile`)